packages/common/src/components/utils/Modal/TextWithInfoModal.tsx

```diff
@@ -16,10 +16,9 @@ interface Props {
   text: string
   modalTitle: string
   className?: string
-  additionalStyles?: string
 }
 
-const TextWithInfoModal: React.FC<Props> = ({ text, modalTitle, children, additionalStyles, className }) => {
```

Comment:
This can't be removed since the component is used in other places in the code which uses this attribute. If you feel the need to remove it you need to refactor the other components.

Reply:
I removed it since it is not used anywhere in the component. It's just imported as a prop and then not used. I had a quick look throw where the component is used and additionalStyles was only used in one place so I refactored that code to be className instead.

Reply:
It looks like the reason why it is not used in the component is because it wrongly removed when the component was refactored from a link to a button. When we change code which is not realted to the Jira we need to make sure that we don't remove functionality. Otherwise we will create hidden regression bugs since test won't look at the code which is not related to the Jira.